.vortex/installer/src/Utils/OptionsResolver.php

```
@@ -122,7 +122,7 @@ public static function resolve(array $options): array {
     }
 
     // Internal flag to skip processing of the demo mode.
-    $config->set(Config::IS_DEMO_DB_DOWNLOAD_SKIP, (bool) Env::get(Config::IS_DEMO_DB_DOWNLOAD_SKIP, FALSE));
+    $config->set(Config::IS_DEMO_DB_FETCH_SKIP, (bool) Env::get(Config::IS_DEMO_DB_FETCH_SKIP, FALSE));
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'DrevOps\VortexInstaller\Utils\Env' in method 'resolve'. (undefined)

(StaticAccess)